volume_linux.go

```diff
@@ -26,17 +29,24 @@ func getVolumeCmd() []string {
 	if useAmixer {
 		return []string{"amixer", "get", "Master"}
 	}
-	return []string{"pactl", "list", "sinks"}
+	return []string{"pacmd", "list-sinks"}
 }
 
 var volumePattern = regexp.MustCompile(`\d+%`)
 
 func parseVolume(out string) (int, error) {
 	lines := strings.Split(out, "\n")
+	pa_default_sink_hooked := false
```

Comment:
I would prefer using camelCase in Golang.

Reply:
Fixed. https://github.com/itchyny/volume-go/pull/9/commits/90441c919052855eae2902ed1e06396cbab2c140